pkg/github/types.go

```diff
@@ -9,6 +9,25 @@ type StatusContext struct {
 	PluginName string
 }
 
+const (
+	// EventGUID is sent by Github in a header of every webhook request.
+	EventGUID = "event-GUID" // aligned with Prow fields
+	// Event is sent by GitHub in a header of every webhook request.
+	Event = "event-type" // aligned with Prow fields
```

Comment:
Shouldn't that be named `EventType`?

Reply:
Only if another `EvenType` struct wouldn't be just below ;)